bin/utils/ensure-up-to-date

```diff
@@ -53,6 +53,8 @@ if [ -n "$(git status --porcelain)" ]; then
     git --no-pager diff
     echo "Perform git status"
     git status
+    echo "This runs in pull requests against the anticipated merge commit (as if the PR was merged now)."
+    echo "When you see unexpected files here, it likely means there are newer commits in master to pull into your branch."
```

Comment:
How about
it likely means that there are newer commits in master that you need to include. Please rebase or merge master into your branch and run bin/utils..

Reply:
That last part is redundant with the line right after my change:

```
Please run 'bin/utils/ensure-up-to-date' locally and commit changes (UNCOMMITTED CHANGES ERROR)
```

:)